index.html

```diff
@@ -241,6 +241,27 @@ <h2>Contact</h2>
 
         If you need a invitation letter for a visa for Germany, please feel free to contact us.
       </section>
+
+      <section id="sponsor-community">
```

Comment:
issue: should also be added to the `sponsors.html` file

Reply:
added